### ISSUE TITLE
invalid path in windows

### ISSUE THREAD

**Issue by podhmo** [podhmo/individual-sandbox#2]

**Comment by podhmo** [comment]
```
error: invalid path 'daily/20171225/example_reqtrace/outdated/0000get_https:__pypi.python.org_simple_zope-interface_.json'
fatal: Could not reset index file to revision 'remotes/origin/master'.
```

**Comment by podhmo** [comment]
fixed by #2776148459fdd1accc6e16b6d3e718d06206557d
